src/components/PackageList/PackageList.tsx

```diff
@@ -1,4 +1,4 @@
-import React, { Fragment, ReactElement } from 'react';
+import * as React from 'react';
```

Comment:
We usually uses `import React, { Fragment, ReactElement } from 'react';` format, this is for consistence and using `*` is not a good practice in general.

Reply:
@juanpicado  understood I will follow the former pattern for future PRs. Out of curiosity what makes `*` not good practice? is it just cause we are importing everything?